test/unit/alphabet/composite/alphabet_variant_test.cpp

```diff
@@ -332,3 +333,31 @@ TEST(alphabet_variant_test, two_different_variants)
     seqan3::alphabet_variant<seqan3::rna4, seqan3::gap> r2{'A'_rna4};
     EXPECT_EQ(l, r2); // this works because rna4 and dna4 are implicitly convertible to each other
 }
+
+TEST(alphabet_variant_test, char_is_valid_for)
```

Comment:
It would be really awesome if we could use TYPED_TEST here, we would only need to somehow extract all bases.

EDIT://

Oh, we have that `::seqan3_required_types`, which is a `type_list<alphabets...>`

Maybe we should make this a public property :thinking:

Reply:
this can be part of another PR?